controllers/postComment.controller.js

```diff
@@ -0,0 +1,16 @@
+const { insertComment } = require('../models/postComment.model')
+
+exports.postComment = (req, res, next) => {
+
+    const id = req.params.article_id
+    const author = req.body.username
+    const body = req.body.body
+    
```

Comment:
You can destructure the values like this to make it more readable:
**const {id} = req.params**